CharacterDisplay/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html5>
```

Comment:
Please check the doctype, for html 5 it should be <!doctype html> but not <!doctype html5>

Reply:
And verify in all html pages

Reply:
changes are done